sources/packages/backend/apps/api/src/route-controllers/models/pagination.dto.ts

```diff
@@ -81,8 +81,21 @@ export class InstitutionUserPaginationOptionsAPIInDTO extends PaginationOptionsA
 
 export class ConfirmationOfEnrollmentPaginationOptionsAPIInDTO extends PaginationOptionsAPIInDTO {
   @IsOptional()
-  @IsIn(["fullName", "applicationNumber", "disbursementDate", "coeStatus"])
+  @IsIn([
+    "fullName",
+    "applicationNumber",
+    "disbursementDate",
+    "coeStatus",
+    "studyEndDate",
+  ])
   sortField?: string;
+
```

Comment:
Can we follow the majority of classes in this file and remove the blank lines between the properties?